.github/pull_request_template.md

```diff
@@ -30,18 +27,19 @@ Fixes #
 
 ## Previewing Changes
 
-<!--- Delete this section if there are no user visible changes. --->
-<!--- List out the steps needed to preview the user visible changes. --->
+<!--- DELETE THIS SECTION IF THERE ARE NO VISIBLE CHANGES. --->
+
+<!--- DETAIL the steps needed to preview your user visible changes. --->
 <!--- Be very specific about what to look for and what things to try. --->
 
-1. Click the `https://s4us-pr-NUMBER.onrender.com` URL **render[bot]** left in the comment below.
-2.
+<!-- TODO: REPLACE "NUMBER" WITH THE PULL REQUEST NUMBER --->
+<!-- The URL should match the URL left by render[bot]'s comment. --->
+1. Go to https://s4us-pr-NUMBER.onrender.com.
```

Comment:
I think this is what is making the test fail.

Reply:
Dang yeah you're right. Hmm, I need to make a two line change to get this to work. This PR is low priority so I think I'm going to work on that PR that auto-approves/keeps approval for small changes after an approval.